url.bs

```diff
@@ -2570,20 +2566,20 @@ then runs these steps, returning an <a>ASCII string</a>:
 <h3 id=url-equivalence>URL equivalence</h3>
 
 <p>To determine whether a <a for=/>URL</a> <var>A</var>
-<dfn export for=url id=concept-url-equals lt=equal>equals</dfn> <var>B</var>, optionally with an
-<i>exclude fragments flag</i>, run these steps:
+<dfn export for=url id=concept-url-equals lt=equal>equals</dfn> <var>B</var>, with an optional
+boolean with an <dfn export for="url/equals"><var>exclude fragments</var></dfn> (default false), run
+these steps:
 
 <ol>
  <li><p>Let <var>serializedA</var> be the result of <a lt="URL serializer">serializing</a>
- <var>A</var>, with the <i>exclude fragment flag</i> set if the
- <i>exclude fragments flag</i> is set.
+ <var>A</var>, with <a for="URL serializer"><var>exclude fragment</var></a> set to
```

Comment:
We kind of settled on `<i>` at call sites instead of `<var>`, although it mostly only matters if you're using Bikeshed's `algorithm=""` feature. https://github.com/whatwg/infra/issues/320#issuecomment-692785334 https://github.com/whatwg/infra/pull/325#issuecomment-703493261